src/wasm-linker.h

```diff
@@ -86,18 +89,18 @@ class LinkerObject {
     globls.push_back(name);
   }
 
-  void addRelocation(Relocation::Kind kind, uint32_t* target, Name name, int addend) {
-    relocations.emplace_back(new Relocation(kind, target, name, addend));
-  }
-
-  Relocation* getCurrentRelocation() {
-    return relocations.back().get();
+  void addRelocation(Relocation* relocation) {
+    relocations.emplace_back(relocation);
   }
 
   bool isFunctionImplemented(Name name) {
     return symbolInfo.implementedFunctions.count(name) != 0;
   }
 
+  bool isGlobalImported(Name name) {
```

Comment:
This is backwards compared to `isFunctionImplemented` (i.e. it's like `!isFunctionImported`). Maybe one or the other of these should be switched to make them consistent.